### PR TITLE
Title test too long for mobile 340px view port width - Apply ellipsis text overflow

### PR DESCRIPTION
**Pull request by elvisdorkenoo**
# Title test too long for mobile 340px view port width 

This PR fix the issue 6028, clipping the the header of Task, Contact and Report tabs when the title is too long for mobile view.

# To test this PR
1.  Use a mobile device, browser developer tools, then toggle device to mobile, or set view port width to 340px(more or less)
2.  Go to Task, Contact and/or Report tab
3.  Check if the overflow text of too long titles is replaced by ellipsis (...)

medic/cht-core#6028

# Code review checklist

- [x] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [x] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [x] Tested: Unit and/or e2e where appropriate
- [x] Internationalised: All user facing text
- [x] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by latin-panda — CHANGES_REQUESTED** [review]
Cool! I left a couple of comments inline :)

File: webapp/src/css/inbox.less
Comment (latin-panda) [reply webapp/src/css/inbox.less:1315]: Based on yesterday convo, we won't develop a way to visualise the full title (for now), so you can remove this hover block. :)

Comment (elvisdorkenoo) [reply webapp/src/css/inbox.less:1315]: Yeah, that's correct, thank you. I'll also update the test steps to remove it as well.

**Review by latin-panda — APPROVED** [review]
Nice! 
Please move the ticket to "Ready for AT" and notify QA that this is ready for testing. 
Hopefully we can make it on time for 3.16.x release :)

**Comment by latin-panda** [comment]
@elvisdorkenoo I noticed this PR is ready for merge 🎉 

The following steps are:
- Select "Squash and merge", this way all your commits are compressed into one and allow us to keep the commit history clean.
- Type a meaningful commit message. This is very useful to others when researching about changes in the code and leaves a nice impression from the author 😎 the cherry on the cake. 
```
Ticket: [ link ]

This commit:
- Adds feature A 
- Adds validations to A
- Removes old code that was .......
```
- Move the ticket to "Done"
- Notify the release manager, this case Njuguna, that this ticket is ready to be included in 3.16.x
- 
Thanks!

**Comment by elvisdorkenoo** [comment]
@latin-panda 
Thank you for the tips, I'm on it 👍🏽
Please should we close the issue as well ?

**Comment by latin-panda** [comment]
@elvisdorkenoo I think when you move the issue to "Done", it will close it automatically.
Once the PR is "Squashed and merged" you just need to delete the branch, please! :)

**Pull request merged**
